docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -364,7 +354,9 @@ If using multipass, to find the IP address use the command
 
 ==== Substitute the SDK's docker image and update sample apps
 
-In order to run TH in a machine that uses the 'linux/amd64' platform, you'll need to first build a new SDK docker image.
+If the platform of the machine that will run the TH is 'linux/amr64' it will not be necessary to build a new docker image of the SDK.
+
+To run TH on a machine using the 'linux/amd64' platform, you will need to first build a new SDK docker image.
```

Comment:
How to build a new docker image?

Reply:
The steps are described in the next subsections